flower/views/tasks.py

```diff
@@ -113,8 +116,14 @@ def get(self):
         capp = self.application.capp
 
         time = 'natural-time' if app.options.natural_time else 'time'
-        if capp.conf.timezone:
-            time += '-' + str(capp.conf.timezone)
+        if capp.timezone:
+            # Timezone in IANA format
+            if isinstance(capp.timezone, LocalTimezone):
+                timezone = get_localzone()
```

Comment:
FYI: `str(LocalTimezone())` would give something like `<LocalTimezone: UTC+02>` which is not a valid IANA format.

Reply:
Note that I'm using `lzlocal` because just the UTC offset yields more than one timezone

```python
import pytz
from datetime import datetime

local_now = datetime.now()
local_utc_offset = local_now.astimezone().utcoffset()

matching_timezones = []
for tz_name in pytz.all_timezones:
    tz = pytz.timezone(tz_name)
    if tz.utcoffset(local_now) == local_utc_offset:
        matching_timezones.append(tz_name)

print(matching_timezones)
```

Reply:
interesting